### PR TITLE
docker-quickstart

### PR DESCRIPTION
**Pull request by HuangFru**
add docker module with docker-compose.yml, 3 Dockerfile, and docker-quickstart.md.
Fix spark quickstart docs.

### CODE REVIEW

File: docker/ams/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+FROM openjdk:8u332-jdk
+WORKDIR /usr/local/ams
+RUN wget https://github.com/NetEase/arctic/releases/download/v0.3.0-rc1/arctic-0.3.0-bin.zip \
```
Comment (HuangFru) [inline docker/ams/Dockerfile:4]: 1

File: site/docs/ch/docker-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+cd arctic-0.3.0
+```
+**2.启动AMS**
+
+通过如下命令启动AMS：
+
+```shell
+./bin/ams.sh start
+```
+
```
Comment (baiyangtx) [inline site/docs/ch/docker-quickstart.md:84]: auto start ams in docker

File: site/docs/ch/docker-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,436 @@
+# 使用Docker快速开始
+本指南将用Docker帮助您快速启动并部署AMS(Arctic Meta Service)，Spark，Flink环境，并体验Arctic的各种功能。
+
+- [Docker-Compose](#Docker-Compose)
```
Comment (zhoujinsong) [inline site/docs/ch/docker-quickstart.md:4]: we do not need the menu here, the mkdocs use a defualt menu on the right side of your content

**Review by zhoujinsong — APPROVED** [review]
I have run this quickstary on my local macos env.
It worked well .

**Review by baiyangtx — APPROVED** [review]
fine

**Pull request merged**